swatch-model-events/schemas/event.yaml

```diff
@@ -186,6 +186,7 @@ properties:
       - addon-open-data-hub
       - BASILISK
       - moa-hostedcontrolplane
+      - moa
```

Comment:
As far as I see, we're using an enum Role to parse from string, and then to parse it back to String. 
I think we should remove this Role enum and directly use a string here. wdyt ? @kahowell

Reply:
Thinking we should go with a similar idea as what we did with ProductId. We created a class that only allows values that come directly from our config: https://github.com/RedHatInsights/rhsm-subscriptions/blob/main/swatch-product-configuration/src/main/java/com/redhat/swatch/configuration/registry/ProductId.java

Not required for this fix so I'm approving and can look again if we make updates.

Reply:
I've created https://issues.redhat.com/browse/SWATCH-3002 for futher discussion.